developer-community/tech-talks.md

```diff
@@ -0,0 +1,211 @@
+• Manage Attendees | Emails to Attendees - To be copied from previous events.  
+
+## Meetup.com
+
+Our Meetup.com account details are secretly held by Maximo :)
+
+Each new event, simply duplicate a previous one from our [tech talks group](https://www.meetup.com/es-ES/srcd-tech-talks/). Remember to set the registration as available till the enxt 5min so that by the time people access it, they can no longer register. This is important as to ensure that everyone gets a ticket on Eventbrite and there are no issues on the day of the event.
+
+## Twitter
+• Talks Announcement (after website is ready - speakers list might not be complete)
+Examples [here](https://twitter.com/srcd_/status/836158701618802688) and [here](https://twitter.com/srcd_/status/871647537123479553).
+
+• Announcement of Sale start (speakers list might not be complete)
+Examples [here](), [here](https://twitter.com/srcd_/status/873848034484006912) and [here](https://twitter.com/srcd_/status/874173998715875328).
+
+• Sale Start (speakers list might not be complete)
+Examples [here](https://twitter.com/srcd_/status/864036667002425346), [here](https://twitter.com/srcd_/status/864399082487521280) and [here](https://twitter.com/srcd_/status/874189703444090882).
+
+• Optional: If sold out, waiting list ones
+Examples [here](https://twitter.com/srcd_/status/836876422660620288) and [here](https://twitter.com/srcd_/status/841954400876785665).
+
+• Schedule Speakers ones
+Examples [here](https://twitter.com/srcd_/status/839424717886992384), [here](https://twitter.com/srcd_/status/839782571294752768), [here](https://twitter.com/srcd_/status/864156684482215936) and [here](https://twitter.com/srcd_/status/877066994377519104).
+
+• Schedule Printing materials pics ones
+Examples [here](https://twitter.com/srcd_/status/844951764944240641), [here](https://twitter.com/srcd_/status/845321360507318272) and [here](https://twitter.com/srcd_/status/870684058879012864).
+
+• On the day
+Examples [here](https://twitter.com/srcd_/status/845566900851658753), [here](https://twitter.com/srcd_/status/870922426451984384) and [here](https://twitter.com/srcd_/status/878570613404827648).
+
+# Offline
+## Venue
+At the office:
+Talks are held at Margaret Hamilton and chairs should be set for 75 people. The bathroom on that side needs to be sealed and the only two operational are the ones in the kitchen. Kitchen is where we serve the food and all breaks happen - it should be completely empty and cleaned before and tables put against the wall. Both fridges should only contain things that people can drink/eat. At the door, there should be a table blocking access to the two main rooms in the office so that people go straight to the kitchen.
+
+When the event is outside of Madrid, it's done on a case by case basis but we should definitely look for a place that can host at least 75-100 people. Past editions were held:
+• ML: Kl10ch (Contact: Natalia Kosyanova <nk@kl10.ch>)
```

Comment:
Not sure if this contact info should be public (surely the email with this format should not)